workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/AppOutlet.tsx

```diff
@@ -80,35 +82,62 @@ const AppOutlet = () => {
 	return (
 		<BaseOutlet
 			backTitle={i18n.translate('back-to-my-apps')}
-			routes={({data, placedOrderItems, product}: any) => [
-				{
-					name: i18n.translate('details'),
-					path: '',
-				},
-				{
-					name: i18n.translate('download'),
-					path: 'download',
-					visible:
-						properties.featureFlags?.includes('LPD-21582') &&
-						data?.placedOrder.workflowStatusInfo.code ===
-							ORDER_WORKFLOW_STATUS_CODE.COMPLETED &&
-						placedOrderItems.some(
-							(item: PlacedOrderItems) =>
-								item.virtualItems?.length
-						),
-				},
-				{
-					name: i18n.translate('licenses'),
-					path: 'licenses',
-					visible: !(
-						getProductPriceModel(product).isFreeApp ||
-						(placedOrderItems[0]?.price?.price === 0 &&
-							product?.skus?.some((sku: any) =>
-								isTrialSKU(sku as unknown as SKU)
-							))
-					),
-				},
-			]}
+			routes={({data, placedOrderItems, product}: any) => {
+				function verifyProductType(product: any) {
+					const specification = product?.productSpecifications?.find(
+						(speficication: ProductSpecification) =>
+							speficication?.specificationKey === 'type'
+					);
+
+					const value = specification?.value;
+
+					return value ? value : '';
+				}
+
+				const tabs = [
+					{
+						name: i18n.translate('details'),
+						path: '',
+					},
+					{
+						name: i18n.translate('download'),
+						path: 'download',
+						visible:
+							properties.featureFlags?.includes('LPD-34129') &&
+							verifyProductType(product) !== CLOUD_APP &&
```

Comment:
```suggestion
							productType === ORDER_TYPES.CLOUDAPP
```

Apply the same for the others